index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="assets/style.css">
+    <title>Html Forms</title>
+</head>
+<body>
+
+    <header class="header">
+        <div class="forgot-pwd">
+            <div class="forgot-pwd-text">Already have an intuit account?</div>
+            <a href="">I forgot my user ID or password</a>
+        </div>
+        <button class="signin-button bg-blue"> <i class="fa fa-lock"></i> Sign In</button>
+    </header>
+
+    <main>
+        <section class="container">
+            <div class="flexbox logo-img">
+                <img src="assets/images/logo.png" alt="Company Logo"/>
+            </div>
+
+            <div class="flexbox icon-container"> 
+                <div class="icon-container-logo">
+                    <img src="assets/images/icon-1.png" alt="Mint Icon"/>
+                </div>   
+                <div class="icon-container-txt">mint</div>
+
+                <div class="icon-container-logo">
+                    <img src="assets/images/icon-2.png" alt="Quickbook Icon"/>
```

Comment:
change `src="assets/images/icon-2.png"` to:

- `src="assets/images/icon-2.PNG"`